lib/lists.nix

```
@@ -1792,6 +1795,41 @@ rec {
   */
   unique = foldl' (acc: e: if elem e acc then acc else acc ++ [ e ]) [];
 
+
+  /**
+    Removes duplicate strings from the `list`. O(n) complexity.
```

To match the documentation suggestion above.

```suggestion
    Removes duplicate strings from the `list`. O(n log n) complexity.
```